src/engine/renderer/tr_surface.cpp

```diff
@@ -1257,6 +1262,14 @@ void Tess_SurfaceIQM( srfIQModel_t *surf ) {
 			TransInsScale( modelScale, bone );
 		}
 	}
+	else
+	{
+		for ( ; bone < lastBone; bone++ )
```

Comment:
These two cases can be merged. `TransInit(bone); TransAddScale(entityScale, bone);` is equivalent to `TransInitScale(entityScale, bone);`.

Reply:
Oh, nice, and now you say that's the same, I notice I did the commit “implement iqm skipTangentSpace alternative like md5” absolutely wrong: on that part I imported what was already done instead of what was not done yet. That makes me notice I had another `TransInitRotationQuat( modelBone->rotation, bone ); TransInverse( &modelBone->joint, bone );` occurrence to optimize in ` precompute md5 bone transform like iqm`. That also means I can do `TransInitScale(entityScale, bone);` in md5 code as well. Impressive sight!